### ISSUE TITLE
Client-only field depending on query

### ISSUE THREAD

**Issue by rhlsthrm** [gqless/gqless#97]
This tool is amazing, but some of the docs are hard to parse. I'm trying to define a client-only field that runs a query and uses that result to generate the field.

It looks like I would add an export to the `extensions` generated directory's `index.ts`. For a simple example, let's use this sample graphql endpoint: https://lucasconstantino.github.io/graphiql-online/.

Say I want to add a field to the `User` object called `nameLength` that queries the name and counts the number of letters in the name. How would I define the User object to run the query on itself?

**Comment by PabloSzx** [comment]
Hi @rhlsthrm, We have just released the new version v2 of gqless, please visit https://gqless.com for instructions, it now has Mutation support, Subscriptions support, Normalization and much more, feel free to enter our [discord server](https://discord.gg/2gy8ZQxnGk), to open an issue or a discussion about anything related, thanks you for your contribution! 🎉
